EIPS/eip-131.md

```diff
@@ -0,0 +1,85 @@
+---
+eip: 131
+title: Add precompiled BLAKE2b contract
+author: Tjaden Hess (@tjade273), Jay Graber, (@carterpy), James Hancock (@madeoftin)
```

Comment:
There's an extra comma after "Jay Graber".

Reply:
Jay Graber isn't carterypy. He is an author attributed earlier by Tjaden.

Reply:
It seemed redundant to put Carterpy (@carterpy), but I can do that.

Reply:
>  Jay Graber isn't carterypy. He is an author attributed earlier by Tjaden.

Ah! Then it needs some name before the parenthesis otherwise rendering will be weird, such as `carterpy (@carterpy)`

Reply:
Got it, and updated.